velox/common/memory/ByteStream.cpp

```diff
@@ -31,4 +66,55 @@ void ByteStream::flush(OutputStream* out) {
   }
 }
 
+void ByteStream::extend(int32_t bytes) {
+  if (current_) {
+    current_->fill = current_->position;
+  }
+  if (current_ && current_ != &ranges_.back()) {
+    ++current_;
+    current_->fill = current_->position;
+    if (current_->fill < current_->position) {
```

Comment:
This block of code is not clear. We have `current_->fill = current_->position;` on line 75 and then 
`if (current_->fill < current_->position)` on line 76 without any body.